README.md

```diff
@@ -30,8 +30,14 @@ Requirements
 Role Variables
 --------------
 
+Role variables include all supported journald parameters as documented in detail at the [freedesktop.org website](https://www.freedesktop.org/software/systemd/man/journald.conf.html#).
```

Comment:
Please put the link into the link collection at the bottom of the README (consistency).